testutils/clustermanager/prow-cluster-operation/main.go

```diff
@@ -42,12 +44,16 @@ func main() {
 	}
 	switch {
 	case create:
-		actions.Create(o)
+		err = actions.Create(o)
 	case delete:
-		actions.Delete(o)
+		err = actions.Delete(o)
 	case get:
-		actions.Get(o)
+		err = actions.Get(o)
 	default:
 		log.Fatal("Must pass one of --create, --delete, --get")
```

Comment:
well why not `err = errors.New("Must pass one of --create, --delete, --get")` at this point?

Reply:
Done. Thanks for pointing it out.